app/routes/wow.item-data.$itemId.tsx

```
@@ -99,172 +168,191 @@
 
   if (listing) {
     const now = new Date()
-    const xCategories = listing.priceTimeData.map((_, hoursToDeduct, arr) =>
-      makeTimeString({
-        date: now,
-        hoursToDeduct: arr.length - hoursToDeduct
-      })
+    const xCategories = useMemo(
```

ESLint

[error] React Hook "useMemo" is called conditionally. React Hooks must be called in the exact same order in every component render. Did you accidentally call a React Hook after an early return?

(react-hooks/rules-of-hooks)